examples/solverdummies/c/solverdummy.c

```diff
@@ -23,17 +23,17 @@ int main(int argc, char **argv)
   const char *writeDataName;
   const char *readDataName;
 
-  const char *configFileName  = argv[1];
-  const char *participantName = argv[2];
-
   if (argc != 3) {
-    printf("Usage: ./solverdummy configFile solverName\n\n");
+    printf("The solverdummy was called without arguments. Usage: ./solverdummy configFile solverName\n\n");
```

Comment:
```suggestion
    printf("The solverdummy was called with incorrect arguments. Usage: ./solverdummy configFile solverName\n\n");
```

Reply:
It's actually the number of arguments which is checked here, so maybe I can even get more specific 
```suggestion
    printf("The solverdummy was called with an incorrect number of arguments. Usage: ./solverdummy configFile solverName\n\n");
```